### PR TITLE
added bottom nav to stats page

### PR DESCRIPTION
**Pull request by sep2001aryan**
## Description

> added bottom nav to stats page

## Changes

- changes apply to the stats page 

## Issue

> Closes none

## Impacted Area

> Stats page

## Steps to test

Steps needed to reproduce the scenario from this change to validate if the pull request solve this issue

- start dev server by  cd frontend && npm run dev
- go to http://localhost:8082/#/stats

image

### CODE REVIEW

**Comment by sep2001aryan** [comment]
I am sorry for the delay

File: frontend/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "terminal.integrated.windowsEnableConpty": false
+}
```
Comment (alexanmtz) [inline frontend/.vscode/settings.json:3]: @sep2001aryan this code should not be committed together, it's not related to this PR

Comment (sep2001aryan) [reply frontend/.vscode/settings.json:3]: Should i remove the vscode settings ?

Comment (sep2001aryan) [reply frontend/.vscode/settings.json]: removed it

**Comment by alexanmtz** [comment]
Great @sep2001aryan 👍

**Pull request merged**